docs/Getting-Started.md

```diff
@@ -59,11 +61,12 @@ Fastify offers an easy platform that helps solve all of problems, and more.
 > The above examples, and subsequent examples in this document, default to listening *only* on the localhost `127.0.0.1` interface. To listen on all available IPv4 interfaces the example should be modified to listen on `0.0.0.0` like so:
 >
 > ```js
-> fastify.listen(3000, '0.0.0.0', function (err) {
+> fastify.listen(3000, '0.0.0.0', function (err, address) {
 >   if (err) {
 >     fastify.log.error(err)
 >     process.exit(1)
 >   }
+>   console.log(`server listening on ${address}`)
```

Comment:
Use `fastify.log.info` instead of `console.log`.

Reply:
fixed